_data/compatibility/binance.yaml

```diff
@@ -0,0 +1,67 @@
+---
+name: Binance
+internal_url: /en/compatibility/binance
+logo: /img/compatibility/binance/binance.png
+rbf:
+  tested:
+    date: "2018-11-06"
+    platforms:
+      - web
+    version: "n/a"
+  features:
+    receive:
+      notification: "false"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "false"
+      shows_original_version: "false"
+    send:
+      signals_bip125: "false"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "na"
+      shows_original_version: "na"
+  examples:
+    - image: /img/compatibility/binance/send-screen.png
+      caption: >
+        Sending RBF Transaction - Default send (withdrawal) screen. No RBF options.
+    - image: /img/compatibility/binance/transaction-list-sent.png
+      caption: >
+        Bumping RBF Transactiont - Transaction not sent via RBF. No fee bump options.
+    - image: /img/compatibility/binance/transaction-list-incoming-rbf.png
+      caption: >
+        Receiving RBF Transaction - No unconfirmed transactions appear in transaction list.
+    - image: /img/compatibility/binance/transaction-list-replacement-confirmed.png
+      caption: >
+        Receiving Bumped RBF Transaction - Replacement transaction only shows up after confirmed. Neither transaction shows before.
+segwit:
+  tested:
+    date: "2019-04-11"
+    platforms:
+      - web
+    version: "n/a"
+  features:
+    receive:
+      p2sh_wrapped: "false"
+      bech32: "false"
+      default: "p2pkh"
+    send:
+      bech32: "true" 
+      change_bech32: "untested"
+      segwit_v1: "Does not pass validation"
+      bech32_p2wsh: "false"
+  examples:
+    - image: /img/compatibility/binance/segwit/receive-screen.png
+      caption: >
+        Binance receives via P2PKH addresses. 
+    - image: /img/compatibility/binance/segwit/send-screen.png
+      caption: >
+        Default send screen accepts a bech32 address.
+    - image: /img/compatibility/binance/segwit/send-address-error.png
+      caption: >
+        While Binance allows bech32 withdrawals, you cannot add a bech32 address
+        to your address book _from the send screen_. While attempting to add a bech32 address, after
+        the two-factor authentication code, an address error message appears.
+    - image: /img/compatibility/binance/segwit/address-book-add.png
+      caption: >
+        Bech32 addresses can be successfully added using the address book functionality.
```

Comment:
Did you happen to get any screenshots of the P2WSH addresses not working?  I think that'd be a useful addition here.

Reply:
I went back and tested this again to get screenshots and added one in. Also clarified some of the bech32 p2wsh/p2wpkh distinctions.